testutil/simapp/consumer.go

```diff
@@ -0,0 +1,58 @@
+package simapp
+
+import (
+	appWasm "github.com/CosmWasm/wasmd/app"
```

Comment:
We should add this to `github.com/cosmos/interchain-security/app/wasm-consumer`.

Reply:
We can move app.go to his location, but we will still need to keep reference to github.com/CosmWasm/wasmd in go.mod file.